RadixWallet/Features/AssetTransferFeature/AssetTransfer+View.swift

```diff
@@ -48,8 +50,14 @@ extension AssetTransfer.View {
 
 					FixedSpacer(height: .small2)
 
-					Button(L10n.AssetTransfer.sendTransferButton) {
+					Button {
 						viewStore.send(.sendTransferTapped)
+					} label: {
+						if viewStore.isLoadingDepositStatus {
+							ProgressView()
```

Comment:
hmm, I know I have made the suggestion to put the loading in the Continue button, but I am not sure it is the best as it will not be visible when there are lots of accounts/resources.

Reply:
I think it is fine since most of the times users won't even notice it (as it is too fast). Also, we only block user from moving forward (which is tapping on Continue), so to me it makes even more sense to show up there. If user selected 5 assets, and we display the loading indicator as a centered overlay, then the user won't be able to add the amounts in the meantime.